db/knex_migrations/2024-12-13-1641-add-public-url.js

```diff
@@ -0,0 +1,13 @@
+// Add column publicUrl to monitor table
+exports.up = function (knex) {
+    return knex.schema
+        .alterTable("monitor", function (table) {
+            table.text("public_url", "text");
```

Comment:
I think this file can be removed, right?

Reply:
Yes absolutely, I forgot to remove it, on my way 👌🏻